### DISCUSSION TITLE
General Setup+Help

### DISCUSSION THREAD

**Discussion by anascloud-zb** [microsoft/sample-app-aoai-chatGPT#985] [Q&A]
Hi, 
I have been in the process of making a "copilot" for my team. First we started by uploading documents to an Azure Search Index, have that connected to Azure Open AI, then deploy that playground to copilot studio, which also has been connected to our website. We also originally had vector embeddings made on each doc before upload, but was told by Microsoft that it would not be compatible for our current configuration. That it is not currently supported in copilot studio. Whatever! The process is good using keyword search, except for the citations that are returned which are too ugly for the user experience. After many google searches I found this repo, and it feels like it has everything we need. I set it up on my laptop, adding Azure keys and endpoints as needed, but it just feels like another GPT when I ask it a question (no connection to the search index). Also, is it possible for it to search our website and limit this application to searching only the website and the index? Thank you in advance!

**Comment by jack-vinitsky** [comment]
If your issues with Copilot Studio citations are related to the MS Teams channel.  You can fix them by rendering them in an adaptive card.  I found this [article](https://community.powerplatform.com/_api/msdyn_richtextfiles%2859DD766A-E438-4633-82DC-57F094A1D87B%29/msdyn_imageblob/$value?size=full) to be really helpful.

**Reply by anascloud-zb** [reply]
Hi @jack-vinitsky, appreciate the help. Can you provide a link to this article. The one you gave already is just a picture.

**Comment by jack-vinitsky** [comment]
Regarding connecting this application to your Azure Search Index...you need to set the appropriate Azure Search environment variables as mentioned in the [ReadMe](https://github.com/microsoft/sample-app-aoai-chatGPT#preview-sample-chat-app-with-aoai) of this repo.
